Backends/CLX/clipboard.lisp

```diff
@@ -105,14 +105,32 @@
         (to-type (selection-object-type requested-object))
         (table (climi::get-object-table requested-object)))
     (labels ((wait-for-request ()
-               (or (not (clipboard-outstanding-request port))
-                   (not (eq selection (selection-object-content requested-object)))))
+               (flet ((wait-fn ()
+                        (or (not (clipboard-outstanding-request port))
+                            (not (eq selection (selection-object-content requested-object)))))
+                      (now ()
+                        (/ (get-internal-real-time)
+                           1.0
```

Comment:
Is this to force float coercion? If so, there has to be a cleaner idiom.

Reply:
it is cargo-culted from input.lisp. how about

(/ (float (get-internal-real-time)
   internal-time-units-per-second))

or alternatively:

(/ (get-internal-real-time)
   #.(flaot internal-time-units-per-second))

?